notebooks/run_model_on_snellius.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>I suggest splitting this cell to two parts: one to get the <code>root_path_to_model </code>and another for re-creating the exe file. </p><p> </p><p>The <code>if </code>statement should be removed, because the exe file always exists. If a developer changes  the MATLAB codes, the exe file is re-generated and the old file is re-written. </p><p> </p><p>If the user runs this notebook on a computer node on Snellius, we dont need to submit a bash job inside the notebook. As we are on the node, we can run the <code>mcc</code> command, see <a href="https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/0fddf037438cad715882f452499f88e4d4cba475/exe/build_stemmus_scope_exe.sh#L26" target="_blank">this line</a>. So I suggest using <code>mcc</code> command here to make it clear for users on both Snellius and CRIB. We can add a note that users on Snellius should be able to use MATLAB on Snellius and can load <code>MATLAB/2021a-upd3</code> module here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/processing/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>